chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -363,12 +350,17 @@
   link "/usr/sbin/rccrowbar" do
     action :create
     to "/etc/init.d/crowbar"
-    not_if "test -L /usr/sbin/rccrowbar"
   end
 
-  bash "Enable crowbar service" do
-    code "/sbin/chkconfig crowbar on"
-    not_if "/sbin/chkconfig crowbar | grep -q on"
+  # Make sure that any dependency change is taken into account
+  bash "insserv crowbar service" do
+    code "insserv crowbar"
+    action :nothing
+    subscribes :run, resources(:cookbook_file=> "/etc/init.d/crowbar"), :delayed
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.